tower_cli/utils/resource_decorators.py

```diff
@@ -0,0 +1,54 @@
+# Copyright 2017 Ansible by Red Hat
+# Alan Rominger <arominge@redhat.com>
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import functools
+from tower_cli.utils import types
+
+import click
+
+
+def unified_job_template_options(method):
```

Comment:
Something seems to be not working correctly here. I tested and saw duplicates in help text:
```
...
 -W, --workflow-job-template WORKFLOW
                                  The workflow_job_template field.
  --unified-job-template TEXT     The unified_job_template field.
  -i, --inventory INVENTORY       The inventory field.
  --credential CREDENTIAL         The credential field.
  --job-type TEXT                 The job_type field.
  --job-tags TEXT                 The job_tags field.
  --skip-tags TEXT                The skip_tags field.
  --limit TEXT                    The limit field.
  --inventory-source INVENTORY_SOURCE
                                  Use this inventory source as
                                  unified_job_template field
  --project PROJECT               Use this project as unified_job_template
                                  field
  --job-template JOB_TEMPLATE     Use this job template as
                                  unified_job_template field
  --inventory-source INVENTORY_SOURCE
                                  Use this inventory source as
                                  unified_job_template field
  --project PROJECT               Use this project as unified_job_template
                                  field
  --job-template JOB_TEMPLATE     Use this job template as
                                  unified_job_template field
...
```